CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## Changed
+
+- Updated `KeyProvider` trait to pass through bytes to be encrypted
+- Updated `CachingKeyWrapper` to implement `KeyProvider`
+- Made `aws-kms` and `cache` features on by default
```

Comment:
I'm not sure about making `aws-kms` a default feature. That means to use any other key provider you will need to turn off all default features which feels a bit janky. Is there another reason I'm not seeing for doing this?

Reply:
@coderdan you'll only need to turn off all default features if you want to compile to wasm / use from the browser which is what we have to do for things like tonic already. Since you'd normally expect Rust to be targeting native (or at least this library would since it's not specifically about wasm) I think all these native-only features  are sensible defaults.

This also saves adding `--features aws-kms,cache` to the cargo test in CI as well as remembering to update them every time we add a new feature that we want off in wasm but is fine to be on for native.